misc/docerina/src/main/java/org/ballerinalang/docgen/generator/model/AbstractObjectPageContext.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ballerinalang.docgen.generator.model;
+
+/**
+ * Page context for the Abstract Object page.
+ *
+ * @since 2.0
+ */
+public class AbstractObjectPageContext extends ModulePageContext {
+    public BAbstractObject abstractObject;
```

Comment:
make this a static final constant or private & provide getters if need to access

Reply:
I think this has to be kept public because its accessed by handlebars template